src/common/operational-api-supporting-functions.ts

```diff
@@ -0,0 +1,103 @@
+import axios, { AxiosRequestConfig } from "axios"
+import { AccessToken, EquipmentInfo, EquipmentStatus, ServiceOrder } from "./types"
+
+const API_HOSTNAME = process.env.API_HOSTNAME || 'dev.kone.com'
+const API_EQUIPMENT_ENDPOINT = `https://${API_HOSTNAME}/api/v1/equipment`
+
+/**
+ * Function to fetch basic information of an equipment
+ * @param accessToken valid access token
+ * @param equipmentIdWithPrefix equipment identifier with ken prefix
+ */
+ export async function fetchEquipmentBasicInformation(accessToken: AccessToken, equipmentIdWithPrefix: string): Promise<EquipmentInfo> {
```

Comment:
All these functions are 99% identical. You could combine the with a private helper function that accepts the acessToken, url and error message as parameters.